geomesa-arrow/geomesa-arrow-jts/src/main/java/org/locationtech/geomesa/arrow/jts/impl/AbstractPointVector.java

```diff
@@ -53,8 +53,13 @@ public void set(int index, Point geom) {
       vector.setNull(index);
     } else {
       vector.setNotNull(index);
-      writeOrdinal(index * 2, geom.getY());
-      writeOrdinal(index * 2 + 1, geom.getX());
+      if (swapAxisOrder()) {
```

Comment:
i think the switch away from a map of options to a boolean will fix this, but we don't want to be doing map lookups on each set call.